zk_toolbox/crates/zk_supervisor/src/commands/test/args/revert.rs

```diff
@@ -12,4 +13,6 @@ pub struct RevertArgs {
     pub external_node: bool,
     #[clap(short, long, help = MSG_NO_DEPS_HELP)]
     pub no_deps: bool,
+    #[clap(short, long, help = MSG_NO_KILL_HELP)]
+    pub auto_kill: bool,
```

Comment:
no kill

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2731/commits/1d5c81092c3afb4a58541f696356855d07f7a1c6